src/rgw/rgw_swift_auth.cc

```diff
@@ -682,12 +675,14 @@ void RGW_SWIFT_Auth_Get::execute()
     goto done;
 
   {
-    static constexpr size_t PREFIX_LEN = sizeof("AUTH_rgwtk") - 1;
-    char token_val[PREFIX_LEN + bl.length() * 2 + 1];
-
-    snprintf(token_val, PREFIX_LEN + 1, "AUTH_rgwtk");
-    buf_to_hex((const unsigned char *)bl.c_str(), bl.length(),
-	       token_val + PREFIX_LEN);
+    const char *prefix = "AUTH_rgwtk";
+    constexpr size_t prefix_length = sizeof(prefix) - 1;
```

Comment:
`sizeof(prefix)` here becomes `sizeof(const char*)` instead of `sizeof(char[11])`

consider using `constexpr std::string_view prefix = "AUTH_rgwtk";` and `prefix.size()` instead?

Reply:
I caught this myself on Friday-- I've moved things to [] rather than *, which should correct the problem, but I'm also ok with string_view. I'll have a new patch up soon. Good eyes, thanks!